builtin/providers/aws/resource_aws_route53_zone.go

```diff
@@ -104,3 +111,20 @@ func resourceAwsRoute53Wait(r53 *route53.Route53, ref string) (result interface{
 	}
 	return true, status, nil
 }
+
+func resourceAwsGoRoute53Wait(r53 *awsr53.Route53, ref *awsr53.GetChangeRequest) (result interface{}, state string, err error) {
+
+	status, err := r53.GetChange(ref)
+	if err != nil {
+		return nil, "UNKNOWN", err
+	}
+	return true, *status.ChangeInfo.Status, nil
+}
+
+// CleanChangeID is used to remove the leading /change/
+func CleanChangeID(ID string) string {
```

Comment:
`goamz` includes this method and cleans the `/change/` for us, but `aws-sdk-go` does not. I'm not sure this is the proper place for the function..